.github/workflows/on-demand-vrfv2plus-performance-test.yml

```diff
@@ -25,79 +25,63 @@ on:
-        env:
-          E2E_TEST_CHAINLINK_IMAGE: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink
-          E2E_TEST_CHAINLINK_VERSION: ${{ inputs.chainlink_version }} # Default chainlink version if not overrided in the test config 
-          E2E_TEST_LOKI_TENANT_ID: ${{ secrets.GRAFANA_INTERNAL_TENANT_ID }}
-          E2E_TEST_LOKI_ENDPOINT: https://${{ secrets.GRAFANA_INTERNAL_HOST }}/loki/api/v1/push
-          E2E_TEST_LOKI_BASIC_AUTH: ${{ secrets.GRAFANA_INTERNAL_BASIC_AUTH }}
-          E2E_TEST_GRAFANA_BASE_URL: "http://localhost:8080/primary"
-          E2E_TEST_GRAFANA_DASHBOARD_URL: "/d/ddf75041-1e39-42af-aa46-361fe4c36e9e/ci-e2e-tests-logs"
-          E2E_TEST_GRAFANA_BEARER_TOKEN: ${{ secrets.GRAFANA_INTERNAL_URL_SHORTENER_TOKEN }}
-        with:
-          test_command_to_run: cd ./integration-tests/load && go test -v -count=1 -timeout 24h -run "${{ inputs.test_list_regex }}" ./vrfv2plus
-          test_download_vendor_packages_command: cd ./integration-tests && go mod download
-          test_secrets_override_base64: ${{ secrets[inputs.test_secrets_override_key] }}
-          test_config_override_path: ${{ inputs.test_config_override_path }}
-          aws_registries: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}
-          artifacts_name: vrf-test-logs
-          artifacts_location: ./integration-tests/load/vrfv2plus/logs/
-          token: ${{ secrets.GITHUB_TOKEN }}
-          go_mod_path: ./integration-tests/go.mod
-          should_cleanup: false
-          QA_AWS_REGION: ${{ secrets.QA_AWS_REGION }}
-          QA_AWS_ROLE_TO_ASSUME: ${{ secrets.QA_AWS_ROLE_TO_ASSUME }}
-          QA_KUBECONFIG: ${{ secrets.QA_KUBECONFIG }}
+          TEST_CMD='cd integration-tests/load && go test -v -count=1 -timeout 24h -run "${{ inputs.test_list_regex }}" ./vrfv2plus'
+          TEST_CONFIG_OVERRIDE_PATH=${{ inputs.test_config_override_path }}
+          TEST_TYPE=${{ inputs.performanceTestType }}
+
+          TEST_LIST=$(jq -n -c \
+            --arg test_cmd "$TEST_CMD" \
+            --arg test_config_override_path "$TEST_CONFIG_OVERRIDE_PATH" \
+            --arg TEST_TYPE "$TEST_TYPE" \
+            '{
+              "tests": [
+                {
+                  "id": "TestVRFv2Plus_Performance",
+                  "path": "integration-tests/load/vrfv2plus/vrfv2plus_test.go",
+                  "runs_on": "ubuntu22.04-8cores-32GB",
+                  "test_env_type": "docker",
+                  "test_cmd": $test_cmd,
+                  "test_config_override_path": $test_config_override_path,
+                  "test_env_vars": {
+                    "TEST_TYPE": $TEST_TYPE   
+                  }
+                }
+              ]
+            }')
+
+          echo "test_list=$TEST_LIST" >> $GITHUB_OUTPUT
+
+  run-e2e-tests-workflow:
+    name: Run E2E Tests
+    needs: set-tests-to-run
+    uses: ./.github/workflows/run-e2e-tests-reusable-workflow.yml
+    with:
```

Comment:
lets add 
```
      slack_notification_after_tests: always
      slack_notification_after_tests_name: VRF V2 Plus Performance Tests
```
here as well

Reply:
@iljapavlovs as for `slack_notification_after_tests_notify_user_id_on_failure`, who should it notify on Slack, you? :)

Reply:
> can you add the name of the config here as well? So that its more clear what run it is straight away from Slack message

@iljapavlovs I'll change it to the following title, ok? 

```
slack_notification_after_tests_name: 'VRF V2 Plus Performance Tests (test config: ${{ inputs.test_config_override_path}})'
```

Reply:
added 

```
      slack_notification_after_tests: always
      slack_notification_after_tests_name: "VRF V2 Plus Performance Tests with test config: ${{ inputs.test_config_override_path || 'default' }}"
      slack_notification_after_tests_notify_user_id_on_failure: U03PJF3VDB8
```

Reply:
@iljapavlovs done, please check latest commits